kubetest/main.go

```diff
@@ -418,7 +420,11 @@ func complete(o *options) error {
 func acquireKubernetes(o *options) error {
 	// Potentially build kubernetes
 	if o.build.Enabled() {
-		if err := xmlWrap("Build", o.build.Build); err != nil {
+		err := xmlWrap("Build", o.build.Build)
+		if o.flushMemAfterBuild {
+			flushMem()
```

Comment:
well, ok, xmlwrap the flushMem call then? and maybe just toss the `flushMem` func to util.go

Reply:
Do we want an entry in testgrid for flushMem ???

Reply:
I moved it to util